alacritty_terminal/src/term/mod.rs

```diff
@@ -179,7 +180,17 @@ impl<T> Search for Term<T> {
     }
 }
 
-/// A key for caching cursor glyphs
+/// Cursor storing all information relevant for rendering.
```

Comment:
```suggestion
/// Cursor which is storing all information relevant for rendering.
```

Reply:
The `which is` here would be redundant, omitting it here seems perfectly fine and keeps the comment a good bit shorter without affecting how well it reads. So I think I prefer it without the `which is`.

Reply:
Could we prepend `A` or `The`? The way I read this code is: "What is a `RenderableCursor`? Oh, it is `Cursor storing all information relevant for rendering.`", which doesn't sound quite right. The below comment is also `A key for caching cursor glyphs`, and not `Key for caching cursor glyphs.`

Obviously up to you, but I would like something prepended.

Reply:
We have things like this all over the codebase and it's certainly not uncommon, so I don't feel like it's a good idea to change this based on the existing arguments.

If we should change that, we should change everything and I disagree both with the reasons for this instance and even more with changing all the other instances.

How "you read this code" is fully up to you and it makes no sense to adjust it based on some approaches to how certain people might "read it". What is important is that the existing documentation text by itself reads nice and makes sense and it does.

Just to get into this "the way I read this code" argument a bit more, if you read it like this, it would be perfectly fine:
```
RenderableCursor: Cursor storing all information relevant for rendering
```